CHANGELOG.md

```diff
@@ -17,4 +17,4 @@ We follow the [Semantic Versioning 2.0.0](http://semver.org/) format.
 - Nothing.
 
 ### Fixed
-- Nothing.
+- The question "What school did you attend when you received the loan?" now reads "What school were you attending when you received the loan?"
```

Comment:
Trivial: This could be broken across two lines at 80 char max to avoid wrapping in the editor here.